fs/fs.go

```diff
@@ -49,14 +55,22 @@ type partition struct {
 
 type RealFsInfo struct {
 	partitions map[string]partition
+	labels     map[string]string
```

Comment:
Added. It's from label to source device. We can use the output of this to find the partition info in partitions map.